_data/remote.yml

```diff
@@ -85,6 +85,14 @@ websites:
       - totp
     doc: http://www.cc-onlinehelp.com/en/#index/administrati/users/authenticati/authenticati.html
 
+  - name: MyFRITZ!
+    url: https://sso.myfritz.net/
+    img: myfritz.png
+    tfa:
+      - totp
+      - hardware
+    doc: https://en.avm.de/service/fritzbox/fritzbox-7590/knowledge-base/publication/show/3427
```

Comment:
Is there any other doc page (maybe in German?) that better explains how to enable 2FA?

Reply:
(see below)